src/templates/quickPickConfirmation/text.mjml

```diff
@@ -0,0 +1,10 @@
+Quick {{ user.userFirstName }}!
```

Comment:
@balexander82 Looks like you missed this text version of the quick pick confirmation fyi